### PR TITLE
fix(Core/Creature): Move CREATURE_FLAG_EXTRA_IMMUNITY_KNOCKBACK

### PR DESCRIPTION
**Pull request by Kitzunu**
## Changes Proposed:
-  Rename CREATURE_FLAG_EXTRA_KNOCKBACK_IMMUNE to CREATURE_FLAG_EXTRA_IMMUNITY_KNOCKBACK
-  Change the flag to the correct bit to match TC & out wiki

To find all the entries I used this query from @FrancescoBorzi <3
```sql
SET @flag = 1073741824;

SELECT `entry`, `name`, `flags_extra`
FROM `creature_template` 
WHERE `flags_extra` & @flag = @flag;
```

## Tests Performed:
- Nada

## How to Test the Changes:

1. Not really necessary but knock it back?

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Comment by pklloveyou** [comment]
Creature.cpp
```
-    return cinfo && (cinfo->flags_extra & CREATURE_FLAG_EXTRA_KNOCKBACK_IMMUNE);
+    return cinfo && (cinfo->flags_extra & CREATURE_FLAG_EXTRA_IMMUNITY_KNOCKBACK);
```

File: data/sql/updates/pending_db_world/rev_1610067409942880500.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
...
...6957,36960,36961,36968,36969,36970,36971,36978,36980,36982,37025,37098,37116,37117,37126,37127,37132,37133,37134,37182,37184,37215,37217,37230,37504,37505,37506,37531,37533,37534,37540,37562,37613,37627,37629,37677,37697,37698,37720,37813,37833,37950,37955,37957,37958,37959,37970,37972,37973,38064,38103,38106,38110,38112,38113,38123,38125,38126,38128,38129,38130,38131,38132,38133,38138,38139,38153,38156,38157,38174,38219,38220,38248,38256,38257,38258,38261,38262,38265,38266,38267,38296,38297,38320,38321,38322,38369,38390,38399,38400,38401,38402,38403,38404,38405,38406,38407,38408,38418,38431,38433,38434,38435,38436,38444,38454,38462,38482,38483,38490,38494,38549,38550,38582,38583,38585,38586,38599,38602,38603,38604,38637,38638,38639,38640,38675,38676,38677,38678,38679,38680,38681,38682,38683,38684,38685,38686,38687,38688,38689,38690,38691,38692,38693,38694,38699,38700,38701,38702,38752,38758,38759,38760,38761,38769,38770,38771,38772,38775,38776,38777,38784,38785,39120,39121,39122,39166,39167,39168,39190,39287,39288,39289,39299,39300,39301,39302,39303,39304,39305,39306,39307,39746,39747,39751,39805,39814,39815,39823,39863,39864,39899,39920,39922,39944,39945,40142,40143,40144,40145);
```
Comment (FrancescoBorzi) [inline data/sql/updates/pending_db_world/rev_1610067409942880500.sql:3]: I wonder why not just 
```sql
`flags_extra`=(`flags_extra`&~0x00200000)|0x40000000
```

did you test it?

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1610067409942880500.sql:3]: I have not, does it work?

Comment (FrancescoBorzi) [reply data/sql/updates/pending_db_world/rev_1610067409942880500.sql:3]: I can't test it now, sorry. If you tested your original SQL code then we can keep it like that

**Comment by Kitzunu** [comment]
@pklloveyou thanks!

**Comment by pklloveyou** [comment]
I tested it and it didn't feel wrong.

**Review by FrancescoBorzi — APPROVED** [review]

**Pull request merged**